ddtrace/contrib/tornado/__init__.py

```diff
@@ -0,0 +1,22 @@
+"""
+TODO: how to use Tornado instrumentation
```

Comment:
Example that creates a root span for all handlers (and attaches a ``Context`` in the handler request):
```python
from ddtrace.contrib.tornado.middlewares import TraceMiddleware

class MainHandler(tornado.web.RequestHandler):
    @tornado.gen.coroutine
    def get(self):
        pass

def make_app():
    return tornado.web.Application([
        (r"/", MainHandler),
    ])

if __name__ == "__main__":
    app = make_app()
    http_server = app.listen(8000)
    TraceMiddleware(http_server, tracer, service='mytornado')
    tornado.ioloop.IOLoop.current().start()
```